node/src/rpc/mod.rs

```diff
@@ -26,12 +26,12 @@ use sp_blockchain::{Error as BlockChainError, HeaderBackend, HeaderMetadata};
 use std::sync::Arc;
 
 mod calamari;
-mod common;
 mod dolphin;
+mod manta;
 
 pub use calamari::create_calamari_full;
-pub use common::create_common_full;
 pub use dolphin::create_dolphin_full;
+pub use manta::create_manta_full;
```

Comment:
```suggestion
pub use self::manta::create_manta_full;
```

It will give you a weird error about ambiguous naming